### PR TITLE
Readme edits, fixes #1

### LINKED ISSUE #1: Finish README
My comment for learn

### PR DESCRIPTION
**Pull request by snowtrr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # hello-world
 Learn GIT
+
```
Comment (snowtrr) [inline README.md]: BAD

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+Hello World!
+Alexandr is here!!!
+Eugene is here!!!
```
Comment (snowtrr) [inline README.md]: GOOD!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
 # hello-world
 Learn GIT
+
+Hello World!
+Alexandr is here!!!
```
Comment (genaix) [inline README.md]: good

**Comment by genaix** [comment]
do not merge plzQ!@!!

**Pull request merged**